src/main/java/com/cloudbees/jenkins/support/SupportLogFormatter.java

```diff
@@ -68,7 +68,9 @@ public synchronized String format(LogRecord record) {
                 record.getThrown().printStackTrace(out);
                 out.close();
                 buf.append(writer.toString());
-            } catch (Exception e) {
+            } catch (Error error) {
+                throw error; // We propagate all Errors upstairs
+            } catch (Throwable e) {
```

Comment:
According to the discussion in another thread, such pattern doe not make much sense. I'll cleanup it